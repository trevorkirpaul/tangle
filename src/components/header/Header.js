import React from 'react';

export default () => {
  return (
    <div>
      <p>from header</p>
    </div>
  );
};
