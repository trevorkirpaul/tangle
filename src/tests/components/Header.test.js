import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header/Header';

it('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
