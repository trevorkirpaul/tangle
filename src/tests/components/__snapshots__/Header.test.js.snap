// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<nav>
  <Link
    replace={false}
    to="/"
  >
    <FlatButton
      disabled={false}
      fullWidth={false}
      label="Tangle"
      labelPosition="after"
      labelStyle={Object {}}
      onKeyboardFocus={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchStart={[Function]}
      primary={false}
      secondary={false}
      style={
        Object {
          "margin": "5px",
        }
      }
    />
  </Link>
  <Link
    replace={false}
    to="/createpost"
  >
    <FlatButton
      disabled={false}
      fullWidth={false}
      label="Create Post"
      labelPosition="after"
      labelStyle={Object {}}
      onKeyboardFocus={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchStart={[Function]}
      primary={false}
      secondary={false}
    />
  </Link>
  <Link
    replace={false}
    to="/signout"
  >
    <FlatButton
      disabled={false}
      fullWidth={false}
      label="Sign Out"
      labelPosition="after"
      labelStyle={Object {}}
      onKeyboardFocus={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchStart={[Function]}
      primary={false}
      secondary={false}
    />
  </Link>
</nav>
`;
